api/src/client/AwsMessagingServiceClient.ts

```diff
@@ -0,0 +1,111 @@
+import { InvokeCommand, LambdaClient } from "@aws-sdk/client-lambda";
+import { HealthCheckMetadata, MessagingServiceClient } from "@domain/types";
+import { LogWriterType } from "@libs/logWriter";
+import { ReasonPhrases } from "http-status-codes";
+
+export const AwsMessagingServiceClient = (config: {
+  functionName: string;
```

Comment:
If we're always going to invoke the same function, then this doesn't need to be a param we pass in. It could be hard-coded into the function `sendEmail`